index.html

```diff
@@ -4993,7 +5107,374 @@ <h2><span class="chunk">tIME</span>
+  in other words,
+  it specifies how the output buffer should be changed
+  at the end of the delay (before rendering the next frame).
+  It is encoded as a one-byte unsigned integer.
+</p>
+
+<p>Valid values for `dispose_op` are: </p>
+
+<table class="Regular" summary=
+    "This table defines the disposal operators">
+    <tr>
+    <td class="Regular">0</td>
+    <td class="Regular">`APNG_DISPOSE_OP_NONE`</td>
+    </tr>
+    <tr>
+    <td class="Regular">1</td>
+    <td class="Regular">`APNG_DISPOSE_OP_BACKGROUND`</td>
+    </tr>
+    <tr>
+    <td class="Regular">2</td>
+    <td class="Regular">`APNG_DISPOSE_OP_PREVIOUS`</td>
+    </tr>
+</table>
+
+  <dl>
+    <dt>`APNG_DISPOSE_OP_NONE`</dt>
+    <dd>no disposal is done on this frame before rendering the next; the contents of the output buffer are left as is.</dd>
+    <dt>`APNG_DISPOSE_OP_BACKGROUND`</dt>
+    <dd>the frame's region of the output buffer is to be cleared to fully transparent black before rendering the next frame.</dd>
+    <dt>`APNG_DISPOSE_OP_PREVIOUS`</dt>
+    <dd>the frame's region of the output buffer is to be reverted to the previous contents before rendering the next frame.</dd>
+  </dl>
+
+  <p>If the first <span class="chunk">fcTL</span> chunk uses a `dispose_op` of `APNG_DISPOSE_OP_PREVIOUS` it should be treated as `APNG_DISPOSE_OP_BACKGROUND`. </p>
+
+  <p>`blend_op` specifies whether the frame is to be alpha blended
+    into the current output buffer content,
+    or whether it should completely replace its region
+    in the output buffer.
+    It is encoded as a one-byte unsigned integer.
+  </p>
+
+  <p>Valid values for `blend_op` are:</p>
+
+  <table class="Regular" summary=
+    "This table defines the blend operators">
+    <tr>
+    <td class="Regular">0</td>
+    <td class="Regular">`APNG_BLEND_OP_SOURCE`</td>
+    </tr>
+    <tr>
+    <td class="Regular">1</td>
+    <td class="Regular">`APNG_BLEND_OP_OVER`</td>
+    </tr>
+</table>
+
+<p>If `blend_op` is `APNG_BLEND_OP_SOURCE`
+  all color components of the frame,
+  including alpha,
+  overwrite the current contents of the frame's output buffer region.
+  If `blend_op` is `APNG_BLEND_OP_OVER`
+  the frame should be composited onto the output buffer
+  based on its alpha,
+  using a simple OVER operation as described in
+  <a href="#13Alpha-channel-processing">Alpha Channel Processing</a>.
+  Note that the second variation of the sample code is applicable. </p>
+
+<p>
+  Note that for the first frame,
+  the two blend modes are functionally equivalent
+  due to the clearing of the output buffer
+  at the beginning of each play.
+</p>
+
+<p>
+  The <span class="chunk">fcTL</span> chunk
+  corresponding to the default image,
+  if it exists,
+  has these restrictions:</p>
+
+  <ul>
+    <li>The `x_offset` and `y_offset` fields must be 0.</li>
+    <li>The `width` and `height` fields must equal
+      the corresponding fields from the <span class="chunk">IHDR</span> chunk.</li>
+  </ul>
+
```

Comment:
Oh, you're right. Line 5347 opens the paragraph. Line 5351 closes it.

I was thrown off because the ```<ul>``` on line 5353 is indented as if it was nested inside the paragraph.
I'll mark this comment as resolved.